.travis.yml

```diff
@@ -2,3 +2,9 @@ language: dart
 dart: dev
 script: ./tool/travis.sh
 sudo: false
+branches:
+  only:
+    - master
```

Comment:
Why only test master?  I actually found it handy to have Travis test all branches, since that let us do experimental work on a branch and see the results immediately, without having to do pull requests to trigger testing.